cross/stlink/Portfile

```diff
@@ -4,7 +4,11 @@ PortSystem          1.0
 PortGroup           cmake 1.1
 PortGroup           github 1.0
 
-github.setup        texane stlink 1.3.0
+github.setup        texane stlink 1.5.1
+github.master_sites ${github.homepage}/archive
+distname            v${github.version}
+worksrcdir          ${github.project}-${github.version}
```

Comment:
Are these actually necessary? For example, couldn't the `distname` be handled by doing `github.setup        texane stlink 1.5.1 v` ?

Reply:
Indeed. Changes made.